changelog.d/20240731_163659_yadudoc1729_update_sdk_to_drop_py3_7.rst

```diff
@@ -0,0 +1,6 @@
+Deprecated
+^^^^^^^^^^
+
+- ``globus-compute-sdk`` and ``globus-compute-endpoint`` drops support for Python3.7.
+  Python3.7 reached end-of-life on 2023-06-27, and so will no longer be a test target
+  for our CI/testing systems.
```

Comment:
Should we idly note that Parsl dropped support for 3.7 as well, perhaps with a link to the appropriate change log?

Reply:
Good point, I've added a mention about this.